### DISCUSSION TITLE
please update bitping

### DISCUSSION THREAD

**Discussion by cecepabdul** [engageub/InternetIncome#77] [General]
![bitping](https://github.com/engageub/InternetIncome/assets/34927384/9212108f-99b8-4e0b-9351-b005e54f527c)
![bitping1](https://github.com/engageub/InternetIncome/assets/34927384/d07fb919-7653-4d34-96ad-7450b1c47f95)

**Comment by engageub** [comment]
Hi,
The script always pulls the latest version before starting.

Bitping docker has been updated 9 months ago but they may be updating it internally within the docker container. 
https://hub.docker.com/r/bitping/bitping-node/tags

I see that on my dashboard too but the nodes are online.

Just raised an issue with bitping team on this https://github.com/BitpingApp/Bitping-Node/issues/11

Thank you

**Reply by cecepabdul** [reply]
bitping change to bitpingd https://hub.docker.com/r/bitping/bitpingd

**Reply by engageub** [reply]
Thank you. I see this has been created 4 days ago.
Also, it is beta version. 
No parameters are mentioned. This will be updated soon after verification.

**Comment by engageub** [comment]
Changes have been pushed to test branch. Please find the link below.

https://github.com/engageub/InternetIncome/tree/test

Thank you

**Comment by engageub** [comment]
Changes have been pushed to main branch.
https://github.com/engageub/InternetIncome
Closing this .

**Reply by cecepabdul** [reply]
yay thank you
